test/browser/perf/index.js

```diff
@@ -1,109 +1,77 @@
 import { run } from '@motorcycle/core'
-import { makeDomDriver, input, div, h2 } from '../../../src'
+import { makeDomDriver, h} from '../../../src'
```

Comment:
Just out of curiosity, did you notice if performance was faster with `h` instead of hyperscript-helpers? (the latter wraps the former, so there's at least one more function call in the stack trace)